src/config.rs

```diff
@@ -171,6 +171,11 @@ impl Default for BellStyle {
     fn default() -> Self {
         BellStyle::Audible
     }
+
```

Comment:
Update line 165 with:
`#[cfg(any(windows, target_arch = "wasm32"))]`
And remove this block